package.json

```diff
@@ -48,10 +49,13 @@
     "express": "^4.17.1",
     "helmet": "^4.4.1",
     "nhsuk-frontend": "5.0.0",
+    "nock": "^13.0.11",
     "nunjucks": "^3.2.3",
     "nunjucks-date-filter": "^0.1.1",
     "path": "^0.12.7",
     "serve-favicon": "^2.5.0",
+    "testcafe": "^1.11.0",
+    "testcafe-reporter-nunit": "^0.1.2",
```

Comment:
Shouldn't `nock` and the TestCafe libraries be dev dependencies?

Can you also double-check that the dependencies are up-to-date?

Reply:
I have moved them into devDependencies. I ran 'npm update' but does not seem to have done much so perhaps we are up to date. Is there a flag you typically run it with that I am missing?

Reply:
I think there's a bug in the latest version of npm, which means that it doesn't update `package.json`, only `package-lock.json`. There's an open [issue](https://github.com/npm/cli/issues/708) for it.

When there's only a few I tend to just do `npm install x@latest` for the relevant dependencies – obviously a pain if there are a lot though!

Reply:
Mmm, that is a bad bug, running 'npm outdated' does not seem to return a list either.